internal/compiler-bridge-test/src/test/scala/sbt/internal/inc/ExtractUsedNamesSpecification.scala

```diff
@@ -212,73 +212,76 @@ class ExtractUsedNamesSpecification extends CompilingSpecification {
     ()
   }
 
-  it should "extract sealed classes scope" in {
-    val sealedClassName = "Sealed"
-    val sealedClass =
-      s"""package base
-        |
-        |sealed class $sealedClassName
-        |object Usage extends $sealedClassName
-        |object Usage2 extends $sealedClassName
-      """.stripMargin
+  // This doesn't work in 2.13.0-RC1
```

Comment:
I'm fine disabling this, but a nitpick to reduce noise: can we use `/*` and `*/` instead of `//` please?

Reply:
@eed3si9n Before merging, can you address this nitpick? The rest of the feedback can be handled in another PR